static/sass/config/_color.scss

```diff
@@ -2,60 +2,56 @@
 // ===================
 
 
-
-// Dependencies
-// ------------
-
-@import '~mathsass/dist/math';
-@import '~accoutrement-color/sass/color';
-
-
-
 // Brand Colors
 // ------------
-/// hello, brand colors!
+/// The brand is defined by our primary brand blue,
+/// along with secondary pink and orange base colors.
+/// @group color
 /// @preview color-palette
+/// @type Map
 $brand-colors: (
-  'brand-blue': hsl(195, 85%, 62%),
+  'brand-primary': hsl(195, 52%, 31%),
   'brand-orange': hsl(24, 100%, 62%),
+  'brand-pink': hsl(330, 100%, 45%),
 );
 
+@include herman-add-colors('brand-colors', $brand-colors);
 
 
-// UI Colors
-// ---------
-/// hello, ui colors!
+// Contrast Colors
+// ---------------
+/// The light and dark contrast colors act as
+/// the high and low range of our palette.
+/// @group color
 /// @preview color-palette
-$ui-colors: (
-  'contrast-light': 'brand-blue' ('tint': 90%),
-  'contrast-dark': 'brand-blue' ('shade': 75%),
+/// @type Map
+/// @link http://oddbird.net/accoutrement-color/sassdoc/
+///   Accoutrement Colors
+$contrast-colors: (
+  'contrast-light': #fff,
+  'contrast-dark': 'brand-primary' ('shade': 50%),
 );
 
+@include herman-add-colors('contrast-colors', $contrast-colors);
 
 
 // Theme Colors
 // ------------
-/// hello, theme colors!
+/// Theme colors define broad patterns for
+/// how our colors will be adjusted and applied.
+/// @group color
 /// @preview color-palette
+/// @type Map
 $theme-colors: (
```

Comment:
Just like the last few sites we've been working on, I wonder if "theme" feels a little too generic? Maybe, maybe not. Just a thought.